constants.py

```diff
@@ -637,8 +639,9 @@
 
 kHoodGearRatio = (1 / 5) * (13 / 360)
 
-kHoodMappingFunction = (
-    lambda x: 8.65 - 1.04 * x + 0.268 * x * x
+# 7.69 + -0.0644x + -0.036x^2 + 0.0297x^3
+kHoodMappingFunction = np.poly1d(
```

Comment:
No clue what this is but I trust

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.poly1d.html